frontend/src/pages/Issue/CompanyDetailsForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { Section } from '@digicatapult/ui-component-library'
+import {
+  Input,
+  Button,
+  HeadingText,
+  SmallItallicisedText,
+  ContainerStyle,
+} from '../../components/shared'
+
+export default function CompanyDetailsForm({ handleSubmit, inputs }) {
+  return (
+    <ContainerStyle>
+      <form style={{ textAlign: 'center' }} onSubmit={handleSubmit}>
+        <Section
+          margin={'2px 0px'}
+          headingLevel={1}
+          background={'#FFF'}
+          title={'- NICE Network - '}
+        >
+          {/* <span style={{ fontSize: '8px' }}>Enter details</span> */}
+        </Section>
+        <HeadingText>Welcome to the NICE Network</HeadingText>
+        {/* <Section margin={'2px 0px'} headingLevel={2} background={'#FFF'}> */}
+        <SmallItallicisedText>Your Company Name</SmallItallicisedText>
+        <Input
+          type="text"
+          required
+          name="name"
+          value={inputs.name}
+          placeholder="Company name"
+        />
+        <SmallItallicisedText>
+          Enter your Companies House Number
+        </SmallItallicisedText>
+        <Input
+          required
+          type="text"
+          pattern="^[a-zA-Z0-9]{8}$"
+          name={'houseNo'}
+          value={inputs.houseNo}
+          placeholder="Company house no."
+        />
```

Comment:
the 'Please match...' alert could be centered or take up the full width, and show what the requested format should be?
<img width="413" alt="image" src="https://github.com/digicatapult/nice-agent-portal/assets/40722025/bd8661ff-e298-4515-9179-c630d5b2d928">